content/visualisation/visualisation_1.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Example Table and Visualisation"
+post = ""
+weight = 1
++++
+
+Below is an example of a fractional attribution report table, which shows the calculated ROAS by attribution channel. 
+
+| conversionWindowStartDate | conversionWindowEndDate | channel           | conversions | revenue | spend  | roas |
+| ------------------------- | ----------------------- | ----------------- | ----------- | ------- | ------ | ---- |
+| 2022-06-03                | 2022-08-01              | Direct            | 699.8       | 137050.5| 10000.0| 13.7 |
+| 2022-06-03                | 2022-08-01              | Organic_Search    | 269.5       | 23292.3 | 10000.0| 2.33 |
+| 2022-06-03                | 2022-08-01              | Paid_Search_Other | 50.4        | 4875.5  | 10000.0| 0.49 |
+| 2022-06-03                | 2022-08-01              | Display_Other     | 21.3        | 2069.0  | 10000.0| 0.21 |
+| 2022-06-03                | 2022-08-01              | Referral          | 12.4        | 653.3   | 10000.0| 0.07 |
+| 2022-06-03                | 2022-08-01              | Unmatched_Channel | 4.08        | 544.6   | 10000.0| 0.05 |
+| 2022-06-03                | 2022-08-01              | Video             | 1.5         | 29.5    | 10000.0| 0.003|
+
+This table shows the ROAS figures for each channel during the conversion window specified. It also shows the conversions attributed to each channel, and the revenue associated with that channel. In our example we have a uniform spend ($10000) for each channel but you can substitute and join this to your own internal dataset to accurately account for spend and ROAS for your own marketing budgets, see [Next steps](/accelerators/fractribution/next_steps/next_steps_1/).
+
+***
+
+For comparison, below is an example of the output of running a traditional last-touch attribution model on the same sample data - [example instructions here](https://docs.snowplow.io/docs/tutorials/tutorial-first-and-last-touch-attribution/). The arrows beside the values indicate the direction of the difference in attribution calculated.
+
+| conversionWindowStartDate | conversionWindowEndDate | channel           | conversions | revenue    | spend  | roas    |
+| ------------------------- | ----------------------- | ----------------- | ----------- | ---------- | ------ | ------- |
+| 2022-06-03                | 2022-08-01              | Direct            | 687  ↓      | 121247.1 ↓ | 10000.0| 12.1  ↓ |
```

Comment:
Probably good to put the arrows in brackets to differentiate them from the value. Might be worth trying them before the value in each cell as well, so that the arrows are all lined up in a uniform way (yes it is a bit OCD of me).

Reply:
No worries - have updated these now.